doc/documentation/src/analysis_guide_templates/preprocessing.rst

```diff
@@ -6,15 +6,28 @@ Preprocessing
 |FOURC| reads the mesh, boundary conditions, materials and simulation parameters from a central
 input file.
 
-.. admonition:: Under development
 
-    A large refactoring effort is currently in progress to improve |FOURC|'s input files.
-    The information in this section is not fully updated to reflect these changes.
+Working with |FOURC| input files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-There are not so many means to create a valid input file. At this point, we know of the following
-different ways to create an input file. In general, you'll have two options:
+|FOURC| input files are text files in the YAML format. Since YAML is a superset of JSON,
+you may also use JSON syntax although we tend to prefer YAML for human-readability.
+By convention, |FOURC| input files have the extension ``.4C.yaml`` or ``.4C.json``.
```

Comment:
Do we actually test files ending with `.4C.json`?

Reply:
No, but I've been meaning to add some for a while. Will do in a new PR.